src/app/templates/app/submit_new_license.html

```diff
@@ -612,6 +613,91 @@
     });
   }
 </script>
+<script type="text/javascript">
+  // Add an event listener to the "Check License Text" button
+  document.getElementById('check-license-btn').addEventListener('click', function() {
+    // Get the license text from the textarea
+    const licenseText = document.getElementById('{{ form.text.id_for_label }}').value;;
+    // const licenseText = "Test-Apache 2.0";
+    
+    // Make an AJAX call to your Django view to match the license text with existing licenses
+    // Replace the URL with the URL of your Django view
+    $.ajax({
+      url: '/app/diff/',
+      type: 'POST',
+      data: {licensetext: licenseText},
+      success: function (data) {
+        console.log("SUCCESS : ", data);
+        var matchType = data.matchType;
+        var matchIds = data.matchIds;
+        if(matchType == "Close match"){
+          $("#modal-header").addClass("yellow-modal");
+          var inputLicenseText = data.inputLicenseText.replace(/\r\n/g,'\n');
+          var originalLicenseText = data.originalLicenseText;
+          var matchingGuidelinesUrl = 'https://spdx.org/spdx-license-list/matching-guidelines'
+          var message = `Close match found! The license closely matches with the license ID(s): <strong>${matchIds}</strong> based on the SPDX Matching guidelines. Press show differences to continue.`
+          $("#modal-header").removeClass("red-modal green-modal");
+          $("#modal-header").addClass("yellow-modal");
+          $(".modal-footer").html(`<a href=${matchingGuidelinesUrl} target="_blank"><button class="btn btn-success btn-space" id="matchingguidelines"><span class="glyphicon glyphicon-link"></span> SPDX Matching Guidelines</button></a><button class="btn btn-success btn-space" id="showDiff"><span class="glyphicon glyphicon-link"></span> Show differences</button>`);
+          $("#modal-body").html(message);
+          $("#myModal").modal({
+            backdrop: 'static',
+            keyboard: true,
+            show: true
+          });
```

Comment:
I think this code is similar to license diff page. Can we have a common util in javascript to have less maintenance ?

Reply:
Ok, I will make a common function for both and will let you know once done

Reply:
Hi @rtgdk , I have created a new file `utils.js` inside which I have combined both of these functions